scripts/zones/Sea_Serpent_Grotto/mobs/Glyryvilu.lua

```diff
@@ -0,0 +1,12 @@
+-----------------------------------
+-- Area: Sea Serpent Grotto
+--  Mob: Glyryvilu
+-- Note: Popped by qm1
+-- !pos 135x -9y 220z
+-- Involved in Quest: An Undying Pledge
+-----------------------------------
+
+
+function onMobDeath(mob, player, isKiller)
+    player:setCharVar("pledgeNM_killed", 1)
```

Comment:
I'm guessing this variable should only be set on players who are currently on the quest.